### ISSUE TITLE
Origins Mod Isn't Compatible With Optifine @Chocohead can u just send me the jar mod in reply

### ISSUE THREAD

**Issue by TraderTheWolf** [Chocohead/OptiFabric#1209]
mods i hav:
origins mod
optifine
optifabric
replay mod
I HAD THE SAME ISSUE WITH REPLAY MOD AND ITS A MIXIN ISSUE AND bluedragon-cairo FIXED IT

**Comment by TraderTheWolf** [comment]
can u send mme the jar @Chocohead also @bluedragon-cairo can u also try fixing the mixin in origins like u did in replay mod?

**Comment by AuroraDoesCode** [comment]
> can u send mme the jar @Chocohead also @bluedragon-cairo can u also try fixing the mixin in origins like u did in replay mod?

Okay so first off, let me be the one to tell you, don’t act so bloody entitled. You can’t just come into the issues section of a github, demand a fix for a mod, and @ the creator to demand said mod download. 

Second, blue dragon didn’t fix the Mixin issue with ReplayMod, he removed the cause of the issue completely. You CANNOT do that for origins, as you can for ReplayMod because of how origins calls it’s code. That’s why there’s a massive section in origins and Apoli support in the compat section of Optifabric. It’s not as easy to ask as it is to actually do the work required to support it, and to be honest, Choco owes you nothing. 

Optifabric is provided as is, as a compatibility shim to replace Minecraft’s standard classes with the ones used by Optifine, and if you have issues with Optifabric not working, take it up with sp614x about why he kept optifine closed source, and made it so invasive with how it does things. Otherwise, stop complaining, or just use Sodium. 

Tl;dr you need to wait along with everyone else for someone to come along and submit a pull request with support. Spamming out the issues section and tagging everyone isn’t a good way to do that.

**Comment by sylv256** [comment]
https://lambdaurora.dev/optifine_alternatives/
